### DISCUSSION TITLE
After uninstall, terminal keeps a command line

### DISCUSSION THREAD

**Discussion by 1qaa** [Homebrew/discussions#4044] [Getting started] [answered]
Not sure if this is the right place to ask this, but after I uninstall homebrew, the terminal window keeps this line, even after exiting/reopening: `-bash: /opt/homebrew/bin/brew: No such file or directory` at the top of the window. I would like for it to not keep it. As far as I'm aware the uninstall was otherwise successful, I used the script provided in the FAQ (`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"`) and then manually deleted the folders it said it couldn't delete at the end.

The uninstall instructions also mentioned being able to run `/bin/bash uninstall.sh --help` which I was kind of confused about, but tried entering after I'd done all the uninstillation, which I think might be the source of the problem? I tried recreating the folders `/bin/brew` to see if that did anything, but it just changed the ending of the message to "is a directory" instead.

I also tried to clear the bash history, but the `~/.bash_history` file was already empty.

This is really not my area of expertise (hence the uninstallation in the first place), so I would really appreciate any help!

**Comment by carlocab** [comment, marked as answer]
Try doing
```sh
open ~/.bash_profile
```
and delete the line that says something like
```sh
eval "$(/opt/homebrew/bin/brew shellenv)"
```
then save the file. If the file is empty, try doing the same with
```sh
open ~/.profile
```

**Reply by 1qaa** [reply]
The second one worked, thank you so much!!
